xml/System.IO.Packaging/CertificateEmbeddingOption.xml

```diff
@@ -19,14 +19,14 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- If the certificate is NotEmbedded in the package, an application that verifies signatures must provide a copy of the certificate in order to verify the signatures that are signed by it.  
+ If the certificate is `NotEmbedded` in the package, an application that verifies signatures must provide a copy of the certificate in order to verify the signatures that are signed by it.  
   
- InSignaturePart adds two informational elements, `<KeyName>` and `<KeyValue>`, as part of the <xref:System.Security.Cryptography.Xml.Signature.KeyInfo%2A> field of the stored digital signature. The `<KeyName>` and `<KeyValue>` elements are not processed as part of signature validation and are therefore not secure from modification. Applications should not make any assumption regarding the validity of these two elements. To avoid undetected modification and possible confusion, applications should use the InCertificatePart option instead of InSignaturePart. The InCertificatePart option does not provide or expose either `<KeyName>` or `<KeyValue>`.  
+ `InSignaturePart` adds two informational elements, `<KeyName>` and `<KeyValue>`, as part of the <xref:System.Security.Cryptography.Xml.Signature.KeyInfo%2A> field of the stored digital signature. The `<KeyName>` and `<KeyValue>` elements are not processed as part of signature validation and are therefore not secure from modification. Applications should not make any assumption regarding the validity of these two elements. To avoid undetected modification and possible confusion, applications should use the `InCertificatePart` option instead of `InSignaturePart`. The `InCertificatePart` option does not provide or expose either `<KeyName>` or `<KeyValue>`.  
   
    
   
 ## Examples  
- The following example shows how to use <xref:System.IO.Packaging.CertificateEmbeddingOption.InSignaturePart> in order to set the <xref:System.IO.Packaging.PackageDigitalSignatureManager>.<xref:System.IO.Packaging.PackageDigitalSignatureManager.CertificateOption%2A> property. For the complete sample, see [Creating a Package with a Digital Signature Sample](https://go.microsoft.com/fwlink/?LinkID=159966).  
+ The following example shows how to use <xref:System.IO.Packaging.CertificateEmbeddingOption> in order to set the <xref:System.IO.Packaging.PackageDigitalSignatureManager>.<xref:System.IO.Packaging.PackageDigitalSignatureManager.CertificateOption%2A> property. For the complete sample, see [Creating a Package with a Digital Signature Sample](https://go.microsoft.com/fwlink/?LinkID=159966).  
   
   
```

Comment:
```suggestion
[!code-csharp[The CertificateEmbeddingOption enumeration](~/samples/snippets/csharp/VS_Snippets_Wpf/PackageDigitalSignature/CSharp/PackageDigitalSignature.cs#SnippetPackageDigSigSign)]
[!code-vb[The CertificateEmbeddingOption enumeration](~/samples/snippets/visualbasic/VS_Snippets_Wpf/PackageDigitalSignature/visualbasic/packagedigitalsignature.vb#SnippetPackageDigSigSign)]
```

Reply:
Oh wow! Didn't notice we were missing the examples here.